tests/contrib/cassandra/test.py

```diff
@@ -264,9 +264,14 @@ def test_batch_statement(self):
         spans = writer.pop()
         eq_(len(spans), 1)
         s = spans[0]
-        eq_(s.resource, 'BatchStatement')
+        eq_(
+            s.resource,
+            (
+                'INSERT INTO test.person_write (name, age, description) VALUES (\'Joe\', 1, \'a\'); '
+                'INSERT INTO test.person_write (name, age, description) VALUES (\'Jane\', 2, \'b\')'
+            )
+        )
         eq_(s.get_metric('cassandra.batch_size'), 2)
-        assert 'test.person' in s.get_tag('cassandra.query')
```

Comment:
actually, should this be `assert s.get_tag('cassandra.query') is None`? to ensure we aren't setting this tag anymore?